### DISCUSSION TITLE
Paste images in editor

### DISCUSSION THREAD

**Discussion by pbdco** [requarks/wiki#5456] [General]
When can we expect this functionality to be available?
Pasting images into the editor should be as simple as CTRL+V :)

**Comment by kobaz** [comment]
If anyone is interested in coding this up. I'll post a bounty $$ for it.  Anyone else want to kick in funding? (Version 2.5.x)

**Reply by vladyxa13** [reply]
Really very useful feature. Did you find anyone?

**Comment by zhuzhzh** [comment]
There is one PR on this.  https://github.com/requarks/wiki/pull/5441
